docs/DeveloperGuide.md

```diff
@@ -1421,7 +1421,117 @@ testers are expected to do more *exploratory* testing.
     1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
        Expected: Similar to previous.
 
-1. _{ more test cases …​ }_
+
+
+### Assignments
+
+### Viewing assignments
+
+1. Viewing assignments with _valid parameters_
+   1. Prerequisites:
+      1. There exists at least 1 student.
```

Comment:
Can be more detailed for the prerequisites. Is it really exist at least 1 student? Or is the index the restricted one?
Say for example I filtered the list to be more specific, will `viewassignments 1` work the same before and after?